pkg/ccl/crosscluster/physical/standby_read_ts_poller_job_test.go

```diff
@@ -54,23 +51,28 @@ func TestStandbyReadTSPollerJob(t *testing.T) {
 	readerTenantName := fmt.Sprintf("%s-readonly", args.DestTenantName)
 	c.ConnectToReaderTenant(ctx, readerTenantID, readerTenantName, 0)
 
+	c.SrcTenantSQL.Exec(t, `
+USE defaultdb;
+CREATE TABLE a (i INT PRIMARY KEY);
```

Comment:
i'm trying to understand what the minimum change necessary to deflake this:
- did we only need to add the `waitUntilReplicatedTime` msg?
- or did we also need to push up this create table stmt as well?